### PR TITLE
Fixed documentation for `sampling_ratio` option

### PR DESCRIPTION
**Pull request by clee1152**
Fixed documentation for `sampling_ratio` option.

### CODE REVIEW

**Review by taylorfturner — APPROVED** [review]

File: docs/source/profiler.rst [hunk 1]
```diff
@@ -600,6 +600,9 @@ Below is an breakdown of all the options.
     * **multiprocess** - Option to enable multiprocessing. Automatically selects the optimal number of processes to utilize based on system constraints.
 
       * is_enabled - (Boolean) Enables or disables multiprocessing
+      
+    * **sample_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.
```
Comment (taylorfturner) [inline docs/source/profiler.rst:604]:
```suggestion
    * **sampling_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.
```

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: docs/source/profiler.rst (hunk 1 above)
Comment (taylorfturner) [reply docs/source/profiler.rst:604]: its actually `sampling_ratio` per https://github.com/capitalone/DataProfiler/pull/845/files#diff-c869332fe6f1bea096dac03badb7365173fa62459962c2e9c020cb53341b1ba6R684

Comment (micdavis) [inline docs/source/profiler.rst:604]:
```suggestion
    * **sample_ratio** - A percentage ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```

Comment (clee1152) [reply docs/source/profiler.rst]: Could you submit the same commit suggestion again? This commit suggestion is outdated after I resolved other commit suggestions.

File: docs/source/profiler.rst [hunk 2]
```diff
@@ -600,6 +600,9 @@ Below is an breakdown of all the options.
     * **multiprocess** - Option to enable multiprocessing. Automatically selects the optimal number of processes to utilize based on system constraints.
 
       * is_enabled - (Boolean) Enables or disables multiprocessing
+      
+    * **sampling_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.
```
Comment (taylorfturner) [inline docs/source/profiler.rst:604]:
```suggestion
    * **sampling_ratio** - A percentage, as a decimal, ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```

File: docs/source/profiler.rst (hunk 1 above)
Comment (taylorfturner) [reply docs/source/profiler.rst]: done

**Review by taylorfturner — APPROVED** [review]

**Comment by taylorfturner** [comment]
FYI @joshuart -- adding docs for your new option of `sampling_ratio`

File: docs/source/profiler.rst [hunk 3]
```diff
@@ -653,7 +656,7 @@ Below is an breakdown of all the options.
       * precision - Finds the precision (significant figures) within the column
 
         * is_enabled - (Boolean) Enables or disables precision
-      * sample_ratio - (Float) The ratio of 0 to 1 how much data (identified as floats) to utilize as samples in determining precision
```
Comment (JGSweets) [inline docs/source/profiler.rst:656]: I think this is a value within `Float`. We should maintain this.

**Review by JGSweets — CHANGES_REQUESTED** [review]

File: docs/source/profiler.rst (hunk 3 above)
Comment (taylorfturner) [reply docs/source/profiler.rst:656]: +1 good catch -- keep this

**Review by taylorfturner — CHANGES_REQUESTED** [review]
just revert the negative back to the original so there is no diff on this

**Review by taylorfturner — APPROVED** [review]
LGTM

File: docs/source/profiler.rst [hunk 4]
```diff
@@ -600,6 +600,9 @@ Below is an breakdown of all the options.
     * **multiprocess** - Option to enable multiprocessing. Automatically selects the optimal number of processes to utilize based on system constraints.
 
       * is_enabled - (Boolean) Enables or disables multiprocessing
+      
+    * **sampling_ratio** - A percentage, as a decimal, ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```
Comment (JGSweets) [inline docs/source/profiler.rst:604]: Just noticed this, correct me if I'm wrong, but i don't think the value can be 0. So as oppose to from 0 to 1, `more than 0 and <= 1`. Not sure best way to type that up though. Does any other doc have a similar to mirror?

Comment (taylorfturner) [reply docs/source/profiler.rst:604]: good catch

**Review by tyfarnan — APPROVED** [review]

**Pull request merged**